base/unicode.jl

```diff
@@ -1,10 +1,8 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 include("unicode/UnicodeError.jl")
-include("unicode/types.jl")
-include("unicode/checkstring.jl")
+# include("unicode/checkstring.jl")
 include("unicode/utf8.jl")
-include("unicode/utf16.jl")
 include("unicode/utf32.jl")
```

Comment:
@StefanKarpinski: Why is `include("unicode/utf32.jl")` not removed here? It looks that the entire file is removed later